src/components/TradeAssetSearch/hooks/useGetCustomTokenPriceQuery.tsx

```diff
@@ -0,0 +1,79 @@
+import { type AssetId, fromAssetId } from '@shapeshiftoss/caip'
+import { isEvmChainId } from '@shapeshiftoss/chain-adapters'
+import { useQueries, type UseQueryResult } from '@tanstack/react-query'
+import axios from 'axios'
+import { mergeQueryOutputs } from 'react-queries/helpers'
+import { useFeatureFlag } from 'hooks/useFeatureFlag/useFeatureFlag'
+
+type UseGetCustomTokenPricesQueryProps = {
+  assetIds: AssetId[]
+}
+
+type UseGetCustomTokenPricesQueryReturn = {
+  assetId: AssetId
+  marketData: ZerionMarketData | undefined
+}[]
+
+const ZERION_BASE_URL = 'https://zerion.shapeshift.com/'
```

Comment:
I think we should add this to the environment variables

Reply:
Stale diff as this needed rebase after #7370 merge, but agree with this!